src/Filter/Filter.php

```diff
@@ -263,13 +264,22 @@ public function addAttribute($name, $value)
 	/**
 	 * @return array
 	 */
-	public function getAttribtues()
+	public function getAttributes()
```

Comment:
ah yeah, probably created branch from that one, since that is merged shouldn't be any problems with this as well :)